packages/clerk-js/src/core/clerk.ts

```diff
@@ -1442,6 +1444,7 @@ export class Clerk implements ClerkInterface {
           generateSignature,
           unsafeMetadata,
           strategy,
+          __experimental_legalAccepted,
```

Comment:
Above we do another `authenticateWithWeb3`. Just confirming this was not overlooked.

Reply:
The above is sign in, right now the legals are only for sign up